js&css/web-accessible/www.youtube.com/appearance.js

```diff
@@ -156,18 +156,22 @@ ImprovedTube.formatSecond = function (rTime) {
 };
 
 ImprovedTube.playerRemainingDuration = function () {
-	document.querySelector('.ytp-time-contents').style.setProperty('display', 'none', 'important');
-	
+	var duration = document.querySelector(".ytp-time-duration").innerText;
+	var remainingDuration = ImprovedTube.storage.player_remaining_duration;
+	if (remainingDuration) {
```

Comment:
For anyone that will review or look at: this condition is important because even if we change the "Show video remaining duration" toggle to OFF, the code will still enter once in this function, so I had to check if the value on storage is true to hide the time duration div, otherwise would not work in real time, as expected.